charts/azure-service-operator/templates/generated/apps_v1_deployment_azureoperator-controller-manager.yaml

```diff
@@ -18,16 +18,6 @@ spec:
         control-plane: controller-manager
     spec:
       containers:
-      - args:
-        - --secure-listen-address=0.0.0.0:8443
-        - --upstream=http://127.0.0.1:8080/
-        - --logtostderr=true
-        - --v=10
-        image: gcr.io/kubebuilder/kube-rbac-proxy:v0.5.0
-        name: kube-rbac-proxy
-        ports:
-        - containerPort: 8443
-          name: https
       - args:
```

Comment:
Unsure if this change is expected, but it looks harmless so signing off.

Reply:
Yeah, I looked at that - as far as I can tell it's just moved the section from one place to another, presumably some non-determinism in the generator?